src/frontend/src/pages/TeamsPage/TeamSpecificPage.tsx

```diff
@@ -40,9 +42,22 @@ const TeamSpecificPage: React.FC = () => {
     </NERButton>
   );
 
+  const setTeamTypeButton = (
```

Comment:
look in other files how you can capitalize this and then call it like \<SetTeamTypeButton\> down where you have the span